content/manager/bootstrapping.md

```diff
@@ -165,6 +165,97 @@ Services:
 {{< /gsHighlight >}}
 
 
+# Deploying a Manager Image
+
+{{% gsNote title="Prerequisites" %}}
+ * A manager image built using the [Image Builder](https://github.com/cloudify-cosmo/cloudify-image-bakery/tree/master/cloudify_marketplace)
+ * For the command-line install, the [`cfy` command]({{< relref "/cli/overview" >}})
+ * Account credentials for the platform you are deploying on
+{{% /gsNote %}}
+
+
+ 1. Create an instance of the [image builder](https://github.com/cloudify-cosmo/cloudify-image-bakery/tree/master/cloudify_marketplace) image
+ 2. Make a note of the IP/hostname
+
+    ```bash
+    $ CLOUDIFY_HOST={your-manager-ip}
+    ```
+
+
+## Command Line
+
+ 1. Switch to the new manager:
+
+    ```bash
+    $ export CLOUDIFY_USERNAME=cloudify
+    $ export CLOUDIFY_PASSWORD=cloudify
+    $ export CLOUDIFY_SSL_TRUST_ALL=true
+    $ cfy use --port 443 --management-ip ${CLOUDIFY_HOST}
+    ```
+
+    {{% gsNote title="TLS" %}}
+    If you have disabled security then use port 80 instead (in this case the `export` lines above are not required): `$ cfy use --port 80 --management-ip ${CLOUDIFY_HOST}`.
```

Comment:
How would they know if security is on or off? They can't disable it.. they need to use an image where security is disabled.. and they need a link to that